django_x509/tests/tests.py

```diff
@@ -0,0 +1,34 @@
+import os
```

Comment:
Regarding tests, I would definitely do it like https://github.com/openwisp/openwisp-firmware-upgrader/pull/60/files